test/clj/rems/test/api/applications.clj

```diff
@@ -819,9 +819,9 @@
         (is (= "workflow/dynamic" (get-in data [:application :workflow :type])))
         (is (= [{:actor user-id
                  :application-id application-id
-                 :event "event/submitted"
-                 :time nil}]
-               (get-in data [:application :dynamic-events])))
+                 :event "event/submitted"}]
+               (->> (get-in data [:application :dynamic-events])
+                    (map #(dissoc % :time)))))
```

Comment:
How about refactoring this test so that it creates its own test data and does not depend on the "common test data"? That would be one step forwards for that issue.

Reply:
It's still an integration test, but maybe it would be possible to set a fixed time using `org.joda.time.DateTimeUtils#setCurrentMillisFixed`